packages/core/data-transfer/lib/providers/local-strapi-destination-provider/restore.ts

```diff
@@ -0,0 +1,31 @@
+import type { ContentTypeSchema } from '@strapi/strapi';
+
+
+const defaultExceptions = ['admin::permission', 'admin::user', 'admin::role', 'admin::api-token', 'admin::api-token-permission', 'plugin::i18n.locale']
+
+export type DeleteOptions = {
+    exceptions?: string[],
+    conditions?: {
+        [contentTypeUid: string]: {
+            where: any
+        }
+    }
+}
+
+export const deleteAllRecords = async (strapi: Strapi.Strapi, deleteOptions?: DeleteOptions) => {
+    const conditions = deleteOptions?.conditions ?? {};
+    const exceptions = deleteOptions?.exceptions ?? defaultExceptions;
```

Comment:
well, I wouldn't say add on top, because we want to have the option to delete everything
As for users passing exceptions, maybe we can do it in the opposite direction and pass what we want to delete instead and have a default delete instead. I think it will make things easier moving forward once we want to make it more flexible

Reply:
And anyways for the users they will not be passing what they want or not want to delete, they will just be at first import everything and then in the future they will probably allow partial imports. and the deletion will be something we handle behind the scenes